test/test-rpm-macros.sh

```diff
@@ -131,13 +131,17 @@ done
 # - *_create_package macros do work correctly
 # - shell syntax is correct (https://github.com/systemd/systemd/commit/93406fd37)
 # - RPM macros, loaded from macros.in, are actually expanded
+
+# if a build directory is specified, add it to pkgconfig search path
```

Comment:
The build directory is always specified, I think. See the beginning of the script. Could you update the comment?